client/packages/common/src/intl/locales/en/common.json

```diff
@@ -695,5 +695,23 @@
   "table.select-unselect-all-columns": "Select / unselect all columns",
   "table.show-columns": "Show / hide columns",
   "title.confirm-delete-encounter": "Delete encounter",
-  "warning.caps-lock": "Warning: Caps lock is on"
+  "title.details": "Details",
+  "title.months-of-stock": "Months of Stock",
+  "title.new-sensor": "New sensor added",
+  "title.packaging": "Packaging",
+  "title.patient-retrieval-modal": "Patient Data Retrieval",
+  "title.pricing": "Pricing",
+  "title.repack-complete": "Repack complete",
+  "title.repack-details": "Repack Details",
+  "title.sensor-details": "Sensor Details",
+  "title.stock-line-details": "Stock Line Details",
+  "title.stock-on-hand": "Available Stock",
+  "tools": "Tools",
+  "tooltip.import-fridge-tag": "Import Berlinger Fridge-tag and Q-tag log files. A sensor will be created, if necessary, for each log file imported.",
+  "warning.cannot-create-placeholder-packs": "There is a total of {{allocatedQuantity}} packs available. Unable to allocate all {{requestedQuantity}} packs.",
+  "warning.cannot-create-placeholder-units": "There is a total of {{allocatedQuantity}} units available. Unable to allocate all {{requestedQuantity}} units.",
+  "warning.caps-lock": "Warning: Caps lock is on",
+  "warning.field-not-parsed": "{{field}} not parsed",
+  "warning.nothing-to-supply": "Nothing left to supply!",
+  "messages.loading-report": "Loading report..."
```

Comment:
I think this is the only line we added right? Can you remove the others? Not a major as I assume they've been added in another branch?